# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccUtils.cpp
# Commit: 3260905f68d1
# Full Hash: 3260905f68d15d2991157be3123ec164537892c5
# Author: James Teh <jteh@mozilla.com>
# Date: 2020-06-02 21:53:59
# Description:
#   Bug 1642480: Null check content in nsAccUtils::IsARIALive. r=MarcoZ
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77748
# ==============================================================================

diff -r 5bbbf98b5a9c -r 3260905f68d1 accessible/base/nsAccUtils.cpp
--- a/accessible/base/nsAccUtils.cpp	Tue Jun 02 05:53:52 2020 +0000
+++ b/accessible/base/nsAccUtils.cpp	Tue Jun 02 05:56:22 2020 +0000
@@ -474,6 +474,9 @@
   // This should be the same as the container-live attribute, but we don't need
   // the other container-* attributes, so we can't use the same function.
   nsIContent* ancestor = aAccessible->GetContent();
+  if (!ancestor) {
+    return false;
+  }
   dom::Document* doc = ancestor->GetComposedDoc();
   if (!doc) {
     return false;
